corehq/apps/hqwebapp/migrations/0011_auto_20231010_0950.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.20 on 2023-10-10 09:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0010_maintenancealert_scheduling'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='maintenancealert',
+            name='created_by_domain',
+            field=models.CharField(db_index=True, max_length=255, null=True),
+        ),
+        migrations.AddField(
+            model_name='maintenancealert',
+            name='created_by_user',
+            field=models.CharField(max_length=128, null=True),
+        ),
+        migrations.AlterModelTable(
+            name='maintenancealert',
+            table='hqwebapp_maintenancealert',
+        ),
+        migrations.RenameModel(
+            old_name='MaintenanceAlert',
+            new_name='CommCareHQAlert',
+        ),
+    ]
```

Comment:
Just to share what this would run
```
$ ./manage.py sqlmigrate hqwebapp 0011
corehq.util.log.HqAdminEmailHandler logger is being changed to logging.StreamHandler
BEGIN;
--
-- Add field created_by_domain to maintenancealert
--
ALTER TABLE "hqwebapp_maintenancealert" ADD COLUMN "created_by_domain" varchar(255) NULL;
--
-- Add field created_by_user to maintenancealert
--
ALTER TABLE "hqwebapp_maintenancealert" ADD COLUMN "created_by_user" varchar(128) NULL;
--
-- Rename table for maintenancealert to hqwebapp_maintenancealert
--
--
-- Rename model MaintenanceAlert to CommCareHQAlert
--
CREATE INDEX "hqwebapp_maintenancealert_created_by_domain_c68ad7b8" ON "hqwebapp_maintenancealert" ("created_by_domain");
CREATE INDEX "hqwebapp_maintenancealert_created_by_domain_c68ad7b8_like" ON "hqwebapp_maintenancealert" ("created_by_domain" varchar_pattern_ops);
COMMIT;
```